src/laundryScraper.py

```diff
@@ -1,152 +1,135 @@
+                    machineType = 'washer'
+                elif str(findMachineType).find('dryer') != -1:
+                    machineType = 'dryer'
+                
+            if row.find('td', 'bgdesc'):
+                machineNumFound = True
+                machineNum = row.find('div','desc').get_text().strip()
+                
+                if machineNum == '':
+                    countMachineNotFound += 1
+                    machineNum = 'notFound{}'.format(countMachineNotFound)
+                
+                
+            if row.find('div', 'runtime'):
+                statusFound = True
+                runTime = row.find('div', 'runtime').get_text().strip()
+                if runTime.find('remaining') != -1:
+                    runTimeInt = [int(s) for s in runTime.split() if s.isdigit()][0]
+                elif runTime.find('out of service') != -1:
+                    runTimeInt = -1
+                elif runTime.find('unknown') != -1:
+                    runTimeInt = -2
+                else:
+                    runTimeInt = 0
+            
+                machineId = buildingId + '-' + machineType + '-' + machineNum
+                if machineId not in foundMachines:
+                    foundMachines.add(machineId)
+                    machineDict[buildingId].append((Machine(runTime, machineId, buildingId, machineType, runTimeInt)))
+
+        """If machine has notFoundX as its number,
+           Iterate through numbers 01 to XX until it finds one that's not taken,
+           and assign it to that machine.
+        """
+        tryMachineNum = 1
+        for machines in machineDict[buildingId]:
+            while machines.machineId.find('notFound') != -1:
+                tryMachineNumStr = str(tryMachineNum).zfill(2)
+                tryMachineId = machines.buildingId + '-' + machines.machineType + '-' + tryMachineNumStr
+                
+                if tryMachineId not in foundMachines:
+                    machines.machineId = tryMachineId
+                    foundMachines.add(tryMachineId)
+                    break
+                else:
+                    tryMachineNum += 1
+        
+        """Using data found above, create Building object
+           Store in buildingList
+        """
+        numWashers = sum(p.machineType == 'washer' for p in machineDict[buildingId])
+        numDryers = sum(p.machineType == 'dryer' for p in machineDict[buildingId])
+        buildingList.append(Building(buildingId, buildingInfo['friendlyNames'], numWashers, numDryers))
+        
+    """Call writeDynamo to re-format buildingList and machineDict and write to Dynamo table"""
+    writeDynamo(buildingList, machineDict)
+    
+"""Iterate through buildingList and machineDict
+   Convert to dynamo format and write info to table
+"""
+def writeDynamo(buildingList, machineDict):
+    """Client with access keys to write to Dynamo tables online"""
+    dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')

```

Comment:
Pretty sure we can actually remove the region name here, and it will automatically choose whatever region the Lambda function is in. That would be good if people want to deploy this in another region.